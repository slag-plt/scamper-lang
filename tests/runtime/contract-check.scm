(string-length (list 1 2 3))

(+ 1 2 3 "bye")

(map char-upcase (list "h" "e" "l" "l" "o"))