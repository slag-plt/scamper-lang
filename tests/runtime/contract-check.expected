
> scamper-lang@1.3.0 driver
> dist/driver/index.js tests/runtime/contract-check.scm

:1:0: Runtime error:
    string-length expected an string? in position 0 but a pair was given
    In program: (string-length (list 1 2 3))
:1:2: Runtime error:
    + expected an number? in position 3 but a lit was given
    In program: (+ 1 2 3 "bye")
:-1:-1: Runtime error:
    char-upcase expected an char? in position 0 but a lit was given
    In program: ([prim charUpcasePrim] "h")
